types/async/index.d.ts

```diff
@@ -1,6 +1,11 @@
-// Type definitions for Async 2.0.1
+// Type definitions for Async 2.4.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
This particular lint rule is fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/31892/commits/539c368b488e72e4d21e35665b6e573e879e0e3c

I have fixed my other rules except for two. This will require more effort and thinking. At the moment I don't have the resources to do that.